### ISSUE TITLE
`enforce-canonical-classes` recommends less optimal way of writing a shortand

### ISSUE THREAD

**Issue by mcmxcdev** [schoero/eslint-plugin-readable-tailwind#320]
### Description

`className="text-lg leading-12"` is recommended to be simplified to `className="text-lg/[2.4000000000000004rem]"` which is not ideal. There seems to be some sort of rounding issue too with it.

<img width="1728" height="534" alt="Image" src="https://github.com/user-attachments/assets/0794dd89-3758-443e-8131-98b3c8dc75ee" />

### Flavor

TSX

### Code Input

```tsx
className="text-lg leading-12"
```

### Expected Behavior

```tsx
Suggest simplification to className="text-lg/12" or at least `className="text-lg/[2.4rem]"`
```

### Actual Behavior

```tsx
See description
```

### ESLint Log

```shell
.
```

### ESLint Config

```typescript
import eslintPluginBetterTailwindcss from "eslint-plugin-better-tailwindcss";

...

{
    plugins: {
      "better-tailwindcss": eslintPluginBetterTailwindcss,
    },
    settings: {
      "better-tailwindcss": {
        entryPoint: resolve(__dirname, "../config-tailwindcss/base.css"),
      },
    },
  },
  {
    rules: {
      "better-tailwindcss/enforce-canonical-classes": "error",
    },
  },
```

### Versions

- ESLint: 9.39.2
- Parser: tseslint.parser
- Plugin: ?
- Node: v24.13.0
